docs/source/en/_toctree.yml

```
@@ -53,8 +53,10 @@
         title: Token classification
       - local: tasks/question_answering
         title: Question answering
-      - local: tasks/language_modeling
-        title: Language modeling
+      - local: tasks/causal_language_modeling
+        title: Causal language modeling
+      - local: tasks/masked_language_modeling
```

You should keep one of the names the same so that the links don't break with previous versions of the doc.
```suggestion
      - local: tasks/language_modeling
```